Prevent json11 files from being installed when Aseprite is installed.
https://github.com/aseprite/aseprite/issues/1602
--- aseprite/third_party/CMakeLists.txt.orig	2025-01-02 08:04:16.000000000 -0600
+++ aseprite/third_party/CMakeLists.txt	2025-01-26 05:14:36.000000000 -0600
@@ -127,7 +127,7 @@
 endif()
 
 # JSON
-add_subdirectory(json11)
+add_subdirectory(json11 EXCLUDE_FROM_ALL)
 
 # libarchive
 set(HAVE_WCSCPY 1)
